moto/awslambda/responses.py

```diff
@@ -359,3 +372,38 @@ def _put_code(self):
             return 200, {}, json.dumps(resp)
         else:
             return 404, {}, "{}"
+
+    def _get_function_concurrency(self, request):
+        path_function_name = self.path.rsplit("/", 2)[-2]
+        function_name = self.lambda_backend.get_function(path_function_name)
+
+        if function_name is None:
+            return 404, {}, "{}"
+
+        resp = self.lambda_backend.get_function_concurrency(path_function_name)
+        return 200, {}, json.dumps({"ReservedConcurrentExecutions": resp})
```

Comment:
The concurrency-attribute might not be set yet, in which case the response would simply be `json.dumps({})`
Would be good to have a test case for this scenario as well

Reply:
Nice catch !!

Reply:
Hey @bblommers . Ran the same request using awscli in debug mode:
![image](https://user-images.githubusercontent.com/24530683/89652132-64268d00-d89b-11ea-9109-53ad62c45482.png)
Looks like it's right :)

Reply:
Fair enough. I ran it using boto3 directly, and then the field isn't returned at all - I guess that's the internal logic in boto that strips None-fields :)